src/main/java/colress/TaskList.java

```diff
@@ -0,0 +1,155 @@
+package colress;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import colress.task.Task;
+
+/**
+ * Represents the TaskList of the Colress chatbot.
+ */
+public final class TaskList {
+    private final ArrayList<Task> tasks;
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+    public Task get(int index) {
+        return tasks.get(index);
+    }
+    public int size() {
```

Comment:
According to the naming conventions, method names should be verbs, perhaps you could call it getSize?